x-pack/plugins/observability_ai_assistant/server/functions/esql/correct_common_esql_mistakes.ts

```diff
@@ -67,12 +67,35 @@ function replaceAsKeywordWithAssignments(command: string) {
   });
 }
 
+function replaceFunctionsinSortWithStats(command: string) {
```

Comment:
I'm not sure if we should do this. This will drop every other column.

Reply:
removed the function and instead improved the system prompt to avoid `SORT COUNT(*)`